packages/web-components/src/drawer-body/drawer-body.styles.ts

```diff
@@ -0,0 +1,41 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '../utils/index.js';
+import {
+  spacingHorizontalM,
+  spacingHorizontalXL,
+  spacingVerticalL,
+  spacingVerticalXXL,
+} from '../theme/design-tokens.js';
+import { typographySubtitle1Styles } from '../styles/partials/typography.partials.js';
+
+/** Drawer styles
+ * @public
+ */
+export const styles = css`
+  ${display('flex')}
+  :host {
+    position: relative;
+    height: 100%;
+    padding: ${spacingHorizontalXL};
+    max-height: calc(100vh - (2 * ${spacingVerticalXXL}));
+  }
+  .drawer-body {
+    display: grid;
+    grid-template-rows: min-content auto min-content;
+    gap: ${spacingVerticalL};
+    overflow-y: auto;
+    max-height: calc(100vh - 2 * ${spacingVerticalL});
+  }
```

Comment:
I think you could remove `.drawer-body` and roll it into the `:host` element maybe. And then maybe use `box-sizing: border-box` to get it the height correct. I'd test this before rolling it in.

```suggestion
  ${display('grid')}
  :host {
    box-sizing: boder-box;
    grid-template-rows: min-content auto min-content;
    position: relative;
    height: 100%;
    padding: ${spacingHorizontalXL};
    max-height: 100svh;
  }
```

I noticed your padding changed from `spacingVerticalXXL` to `spacingVerticalL` so that may be an issue I'm not mentally accounting for.